client_option_insecure.go

```diff
@@ -0,0 +1,14 @@
+package getter
+
+// WithInsecure allows for a user to avoid
+// checking certificates (not recommended).
+// For example, when connecting on HTTPS where an
+// invalid certificate is presented.
+// User assumes all risk.
+// Not all getters have progress support yet.
```

Comment:
```suggestion
// Not all getters have support for insecure mode yet.
```

Looks like this was a copy-paste error from `client_option_progress.go`. But this also points out to me that there's some precedent for having options that don't support all getters, which was one of my initial concerns here. :+1:

Reply:
I pushed a fix for the above, nice catch. I am not a huge fan of having an option that doesn't work everywhere but once I saw the progress bar I thought it would be ok. Also, there aren't good parallels for `insecure` for some of the other getters and I wasn't sure how to handle that.